### ISSUE TITLE
cancelling event with lightning rod

### ISSUE THREAD

**Issue by mordisk10** [GrimAnticheat/Grim#736]
### Describe the false positive and how to replicate it

1. place a block when you are looking to lightning rod
2. cancel `PlayerInteractEvent`
3. now block is not there obviously and grim teleports you back

grim log https://paste.grim.ac/ZzftW (not same with the video but same thing)

demonstration video
[Screencast from 05-08-2022 16:44:00.webm](https://user-images.githubusercontent.com/70017374/183090035-4c5066ec-d8e5-4e8f-9c8c-6931f3cc9f3b.webm)

### Grim version

version is `2.3.21` and 18b5ea8

### Server version

This server is running Jetshine version git-Jetshine-"19e8e02" (MC: 1.19.1) (Implementing API version 1.19.1-R0.1-SNAPSHOT)

basically a Pufferfish fork
(other patches are not related with this for sure)

ViaVersion version 4.4.1
ViaBackwards version 4.4.1

### Plugins

Plugins (10): GrimAC, HarmonyScoreboard, Jailbreak, NBTAPI, PlaceholderAPI, ShinyAPI, ShinyChairs, spark, ViaBackwards, ViaVersion

**Comment by MWHunter** [comment]
Unable to reproduce.
